vm/src/executor.rs

```diff
@@ -77,10 +73,12 @@ impl From<HashingError> for RuntimeError {
 struct Item(Vec<u8>);
 
 impl Item {
+    #[allow(dead_code)]
```

Comment:
I think you'd better to write higher level warning suppression.
You can write `#[allow(dead_code)]` right above the mod executor in `vm/src/lib.rs` or `#![allow(dead_code)]` for crate level suppression. https://doc.rust-lang.org/rust-by-example/attribute.html

Reply:
Thank you for your good idea! I modified it : ))